tests/integrations/mcs/tso/server_test.go

```diff
@@ -152,7 +153,7 @@ func (suite *tsoServerTestSuite) TestPariticipantStartWithAdvertiseListenAddr()
 func TestTSOPath(t *testing.T) {
 	re := require.New(t)
 	checkTSOPath(re, true /*isAPIServiceMode*/)
-	checkTSOPath(re, false /*isAPIServiceMode*/)
+	// checkTSOPath(re, false /*isAPIServiceMode*/)
```

Comment:
Why comment this line?

Reply:
temporary for debugging. I'll restore it.

Reply:
restored